docs/design/extending-api.md

```diff
@@ -113,18 +113,17 @@ For example, if a user creates:
 
 ```yaml
 metadata:
-  name: cron-tab.example.com
+  name: cron-tab.stable.example.com
 apiVersion: experimental/v1alpha1
 kind: ThirdPartyResource
 description: "A specification of a Pod to run on a cron style schedule"
 versions:
-  - name: stable/v1
-  - name: experimental/v2
+- v1
```

Comment:
@caesarxuchao: my change is mostly about declaring having a group there wrong :)

@bgrant0607 OK, I'll add it back.